src/css/map-view.css

```diff
@@ -1410,3 +1410,53 @@ other class: .ui-slider-range */
     }
   }
 }
+
```

Comment:
Compared to layer-category-item, expansion-panel is missing some font/icon size and color treatments. are we saving the polish for later?